app/models/spree/variant_decorator.rb

```diff
@@ -3,6 +3,10 @@ def product_feed_image
     images.where(viewable_type: 'Spree::ProductCatalog').first
   end
 
+  def display_image
+    images.order(:position).first
```

Comment:
i'm usually against default scopes, but would it make sense to default to returning the images by position?

Reply:
Maybe but I felt more comfortable with this more focused change.

Reply:
Originally I had a `FeedProduct` class but that apparently got stripped out last year. I could see bringing that back, too.

Reply:
works for me, wasn't sure how much that would be used and think this line makes it a lot more clear what's actually going on